index.rst

```diff
@@ -302,107 +305,156 @@ Is this case, we could have something like:
+    - It is not possible to load the ``_<site>.yaml`` file (or any file with a ``_`` prefix) using the ``configurationOverride`` attribute
+    - Between this file and the ``_init.yaml`` file, **the configuration must be fully defined**.
 
-    - See this :download:`example _base.yaml <_static/_base.yaml>` for the ATSpectrograph CSC.
+#.  An additional configuration file provides overrides for the configuration parameters set by the previous files.
 
-#.  The labels file (e.g. ``_labels.yaml``) shall continue to exist with the same format and purpose.
-#.  Additional configuration files can provide new values for individual configuration parameters.
+    - See this :download:`configuration parameter override example file <_static/ATSpectrograph_example_config.yaml>` for the ATSpectrograph CSC.
+    - This file is the third configuration file to get loaded by the CSC and will override any previously declared values.
+    - These files are loaded using the ``configurationOverride`` attribute in the ``start`` command
+    - These are not expected to be required as part of regular operations and are meant to be used when a non-standard configuration is required
+    - If an override configuration file is also site specific, then a prefix should be added indicating which site it belongs with (e.g. ``summit_reduced_stage_travel.yaml``)
 
-    - See this :download:`example configuration file <_static/ATSpectrograph_example_config.yaml>` for the ATSpectrograph CSC.
+#.  The labels file, and all notions of labels, shall be deprecated. Only filenames shall be used.
+        - No file shall exist having the name ``default.yaml``.
+          There are other invalid names for files (e.g. ``init.yaml``) which are to be verified by continuous integration tests in the configuration repository.
 
-#.  If a CSC receives a ``start`` command with an empty ``configuration`` (see :ref:`renaming proposal <section-renaming>`) attribute, it shall load the values in ``_base.yaml``.
-#.  If a CSC receives a ``start`` command with a ``configuration`` attribute equal to a label in ``_labels.yaml``, it shall load the values in ``_base.yaml`` first and override those values defined in the mapped configuration file.
-#.  The name ``default`` shall not be used for labels.
-#.  All valid configurations shall have a label to be loaded by the CSC.
-    Configurations cannot be loaded by filename.
-#.  The configuration repository should be kept clean of configurations used for unit testing or for different purposes (e.g. test stand configurations).
-    Configurations tailored for test stand can be kept in different branches/tags.
+#.  If a CSC receives a ``start`` command with an empty ``configurationOverride`` (see :ref:`renaming proposal <section-renaming>`) attribute, it shall load the values in ``_init.yaml`` then the site-specific file (e.g. ``_summit.yaml``).
```

Comment:
One place where we need to note that this is optional.  The following might suffice here: `then the site-specific file (e.g. ``_summit.yaml``), if it exists.`